index.js

```diff
@@ -40,9 +39,13 @@ EncryptedField.prototype.vault = function(name) {
 
             var cipher = crypto.createCipheriv(self._algorithm, self.key, new_iv);
 
-            cipher.end(JSON.stringify(value), 'utf-8');
-            var enc_final = Buffer.concat([new_iv, cipher.read()]);
-            var previous = this.setDataValue(name, enc_final);
+            var enc_final = Buffer.concat([
```

Comment:
this doesn't need to change

Per the docs:
https://nodejs.org/api/crypto.html#crypto_class_cipher

cipher objects are streams and that is where the `.end()` call comes from.